dkist/io/file_manager.py

```diff
@@ -352,11 +358,12 @@ def download(self, path=None, destination_endpoint=None, progress=True, wait=Tru
             )
 
         inv = self._ndcube.meta["inventory"]
+        human_inv = humanize_inventory(inv)
 
         base_path = Path(net_conf.dataset_path.format(**inv))
         destination_path = path or self.basepath or "/~/"
         destination_path = Path(destination_path).as_posix()
-        destination_path = Path(destination_path.format(**inv))
+        destination_path = Path(destination_path.format(**human_inv))
```

Comment:
Not strictly related to this PR, per se, but it might be nice to have some custom error handling if a user tries to interpolate a key that doesn't exist; right now I'm fairly certain it would just throw `KeyError: 'foo'`. Even something like a quick note on how to show available keys would probably help a lot.

Reply:
If there's a new key in the inventory but it's not in the map it just copies it through without translating it, if that's what you mean.

Reply:
Interesting. I think I see. Are you saving if a user has `destination_path = "{datasetID}_{somekey}_data"` then it will interpolate to, e.g., "YAFMS_{somekey}_data"? If so then I'm fine with that.

Reply:
oh no, I was mistaken. You can only use keys which exist.

For Fido there's a list, here there isn't really a way. I think I will open an issue to track that.